tools/barista/src/builder/components.ts

```diff
@@ -73,22 +73,39 @@ function setMetadataDefaults(baristaMetadata: any): BaSinglePageMeta {
   return metadataWithDefaults;
 }
 
+function getPaths(path: string, fileList: string[]): string[] {
```

Comment:
This function should also check for the path provided, not only its sub directories